ts/features/messages/components/Home/homeUtils.ts

```diff
@@ -35,3 +41,31 @@ export const messageListCategoryToViewPageIndex = (
 export const messageViewPageIndexToListCategory = (
   pageIndex: number
 ): MessageListCategory => (pageIndex === 1 ? "ARCHIVE" : "INBOX");
+
+export const accessibilityLabelForMessageItem = (message: UIMessage): string =>
+  I18n.t("messages.accessibility.message.description", {
+    newMessage: message.isRead
+      ? I18n.t("messages.accessibility.message.read")
+      : I18n.t("messages.accessibility.message.unread"),
```

Comment:
```suggestion
newMessage:  I18n.t(`messages.accessibility.message.${message.isRead?"read":"unread"}`)
```



why not use template strings here?

Reply:
Applied!